src/plugins/workspace/server/plugin.ts

```diff
@@ -68,7 +68,7 @@ export class WorkspacePlugin implements Plugin<WorkspacePluginSetup, WorkspacePl
   }
 
   constructor(initializerContext: PluginInitializerContext) {
-    this.logger = initializerContext.logger.get('plugins', 'workspace');
+    this.logger = initializerContext.logger.get();
```

Comment:
Any reason that we remove the params here?

Reply:
the logger have the context already here https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/server/plugins/plugin_context.ts#L196

```typescript
/**
 * Plugin-scoped logger
 */
logger: {
  get(...contextParts) {
    return coreContext.logger.get('plugins', pluginManifest.id, ...contextParts);
  },
},
```